uiowa/uiowa
# Cleanup related to creation of `uiowa_maui` split

- Removes `uiowa_maui` as a dependency from grad and housing site splits along with associated partial config splits
- Moves the LB style for hiding session badges to the `uiowa_maui` feature split from default
- Note: I turned on the `uiowa_maui` for grad and housing in production

# How to test
- Review config changes and confirm that they all make sense
```
ddev blt sync --site grad.uiowa.edu && ddev drush @grad.local uli academics/deadlines
```
- Confirm that page display is unchanged from https://grad.uiowa.edu/academics/deadlines
- Click the "Layout" tab
- Click "Configure" for the existing academic dates block
- Confirm that "Hide session badges" is checked
- Click "Add component" and select the "More" option
- Confirm that "Academic dates" block is available and add the block
- Confirm that "Hide session badges" is available as a style option
```
ddev blt sync --site housing.uiowa.edu && ddev drush @housing.local uli /
```
- Confirm that "Important dates" block display is unchanged from https://housing.uiowa.edu/
- Click the "Layout" tab
- Click "Configure" for the existing academic dates block
- Confirm that "Hide session badges" is checked
- Click "Add component" and select the "More" option
- Confirm that "Academic dates" block is available and add the block
- Confirm that "Hide session badges" is available as a style option
